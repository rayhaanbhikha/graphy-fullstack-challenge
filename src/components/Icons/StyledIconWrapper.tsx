import styled from 'styled-components';

export const StyledIconWrapper = styled.i`
  cursor: pointer;
  margin: 2px;
`