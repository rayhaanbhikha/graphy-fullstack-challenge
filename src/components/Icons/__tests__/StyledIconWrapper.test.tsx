import React from 'react';
import { render } from '@testing-library/react'
import { StyledIconWrapper } from '../StyledIconWrapper';
import 'jest-styled-components';

describe('StyledIconWrapper', () => {

  it('should have correct default styles', () => {
    const { asFragment } = render(<StyledIconWrapper />);
    expect(asFragment()).toMatchSnapshot();
  })
});