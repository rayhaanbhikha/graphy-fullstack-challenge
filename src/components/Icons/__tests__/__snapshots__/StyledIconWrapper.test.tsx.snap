// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledIconWrapper should have correct default styles 1`] = `
<DocumentFragment>
  .c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 3px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  cursor: pointer;
}

.c0:hover {
  background: #b5b5b5;
  border-radius: 2px;
}

<i
    class="c0"
  />
</DocumentFragment>
`;
